shared/oae/css/oae.components.css

```diff
@@ -50,12 +50,18 @@ a.oae-plain-link:hover {
  * Style that can be used to fake focus on an element. This can for example
  * be used when a hidden element needs to provide visual feedback on a different element.
  */
+
 .oae-focus {
     outline: thin dotted #333;
-    outline: 5px auto -webkit-focus-ring-color;
     outline-offset: -2px;
 }
 
+a.oae-focus,
+.btn.oae-focus,
```

Comment:
Do we already have a convention to ignore the `.` when doing alphabetical ordering? It looks funny to me to split the tagName selectors with a className selector. I think I'd prefer:

```css
a.oae-focus,
input.oae-focus,
.btn.oae-focus {
   ...
}
```